### ISSUE TITLE
Hide UI feature

### ISSUE THREAD

**Issue by ghost** [TaktiCool/Streamator#1]
I've been using the streamator today for a little bit and very much like all the functionality it has. The solution of making the ui overlap 10 Pixels on all sides so you can easily crop it is a good and ingenious solution for streamers etc. However i do still feel that a keybind to hide the UI entirely (possibly all UI in arma) would still be benificial. For example in a few missions that i spectated i could't quite read hints,mission objectives and others in the top right as the ui was overlapping them. Also for some people it might prove to be quite a hassle to crop the image every time. Occasions such as screenshots make it so that you ahve to crop the image every time afterwards and you lose a little bit of the resolution. I know it's not much that you lose but it's something and most people doing professional screenshots etc. are very fiddly with that stuff. 

In short the UI is good as it is except maybe a bit more spacing between the keybind hints at the top. But the mod would definitely benefit from the extra keybind to hide the ui entirely on top of it.

**Comment by jokoho48** [comment]
Done in https://github.com/TaktiCool/Streamator/commit/2b55d7402cae1e13de59b24f346123e02abf0dc3
UI can get hidden when Backspace is pressed
